### PR TITLE
Make recently visited websites prominent on the top bar. Avoid favicon cropping. 

### PR DESCRIPTION
**Pull request by fazlerocks**
Before: 
image

After:
image

### CODE REVIEW

**Comment by fazlerocks** [comment]
Maintainers PS: I am not participating in the hacktoberfest. 😆 It's just a coincidence.

File: packages/extension/src/components/DaHeader.vue [hunk 1]
```diff
@@ -281,16 +281,18 @@ export default {
   }
 
   & .header__top-site {
-    width: 24px;
-    height: 24px;
-    border-radius: 50%;
+    width: 28px;
+    height: 28px;
+    border-radius: 4px;
```
Comment (idoshamun) [inline packages/extension/src/components/DaHeader.vue:286]: Let's change the border-radius to be `8px`.
It fits better our style guide.

```suggestion
    border-radius: 8px;
```

**Review by idoshamun — CHANGES_REQUESTED** [review]
Wow, @fazlerocks!
Your contribution is so appreciated. Thank you very much 🙏

**Review by idoshamun — APPROVED** [review]
It's great now! Thank you 🚀

**Pull request merged**